collections/St_Johns_College/SW_V1_St_Johns_College_MS_006.xml

```diff
@@ -0,0 +1,233 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/msDesc/consolidated-tei-schema/master/msdesc.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://raw.githubusercontent.com/msDesc/consolidated-tei-schema/master/msdesc.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xmlns:tei="http://www.tei-c.org/ns/1.0"
+   xml:id="manuscript_12006">
+   <teiHeader xml:lang="en">
+      <fileDesc>
+         <titleStmt>
+            <title>St John's College MS. 6</title>
+            <title type="collection">St John's College MSS.</title>
+            <respStmt xml:id="RM">
+               <resp when="2002">Cataloguer</resp>
+               <persName>Ralph Hanna</persName>
+            </respStmt>
+            <respStmt xml:id="SJC">
+               <resp when="2021">Encoding</resp>
+               <persName>Sian Witherden</persName>
+            </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition>TEI P5</edition>
+            <funder>Conversion of the printed catalogue to TEI funded by <orgName>the Thompson
+                  Family Charitable Trust</orgName></funder>
+         </editionStmt>
+         <publicationStmt>
+            <publisher>Special Collections, Bodleian Libraries</publisher>
+            <address>
+               <orgName type="department">Special Collections</orgName>
+               <orgName type="unit">Bodleian Libraries</orgName>
+               <orgName type="institution">University of Oxford</orgName>
+               <street>Weston Library, Broad Street</street>
+               <settlement key="place_7011931">Oxford</settlement>
+               <postCode>OX1 3BG</postCode>
+               <country>United Kingdom</country>
+            </address>
+            <distributor>
+               <email>specialcollections.enquiries@bodleian.ox.ac.uk</email>
+            </distributor>
+            <idno type="msID">St_Johns_College_MS_6</idno>
+            <idno type="collection">St_Johns_College</idno>
+            <idno type="catalogue">Western</idno>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:lang="en" xml:id="St_Johns_College_MS_6">
+               <msIdentifier>
+                  <country>United Kingdom</country>
+                  <region type="county">Oxfordshire</region>
+                  <settlement key="7011931">Oxford</settlement>
+                  <institution>St John's College</institution>
+                  <repository>St John's College</repository>
```

Comment:
I'm not sure that the repository element is necessary. Perhaps it can be omitted.